src/elements/content-sidebar/activity-feed/task-new/AvatarGroupAvatar.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../common/variables';
+@import './mixins';
+
+$bcs-AvatarGroupAvatar-avatar-size: 32px;
+
+.bcs-AvatarGroupAvatar {
+    display: inline-block;
+    height: $bcs-AvatarGroupAvatar-avatar-size;
+    position: relative;
```

Comment:
I think we needed this somewhere in `Badgeable`. will have a look when there's time.